development.rst

```diff
@@ -52,18 +52,38 @@ Otherwise, to install and run, do:
   $ pip install tox
   $ tox
 
+Running Tests - JavaScript
+--------------------------
+
+You will need `npm <https://www.npmjs.com>`_ installed to run the JavaScript tests.
+Internally, we use `Grunt <https://gruntjs.com>`_ and `QUnit <https://qunitjs.com>`_ to run the tests.
+
+Once ``npm`` is installed, you can install all needed dependencies by running:
+
+.. code-block:: bash
+
+  $ npm install
+
+Run the following to execute the tests:
+
+.. code-block:: bash
+
+  $ npm test
+
 Releasing a new version
 -----------------------
 
 Follow these steps to release a new version of the project:
 
-1. Update your local master with the upstream master (``git pull --rebase upstream master``)
-2. Create a new branch and update ``CHANGES.rst`` with the new version, today's date, and all changes/new features
-3. Commit and push the new branch and then create a new pull request
-4. Wait for tests and reviews and then merge the branch
-5. Once merged, update your local master again (``git pull --rebase upstream master``)
-6. Tag the release with the new release version (``git tag v<new tag>``)
-7. Push the tag (``git push upstream --tags``)
-8. Done. Check `CI <https://github.com/pytest-dev/pytest-html/actions>`_ for release progress.
+1.  Update your local master with the upstream master (``git pull --rebase upstream master``)
+2.  Create a new branch
+3.  Update ``CHANGES.rst`` with the new version, today's date, and all changes/new features
+4.  Update the ``version`` field in  ``package.json`` with the new version
+5.  Commit and push the new branch and then create a new pull request
+6.  Wait for tests and reviews and then merge the branch
+7.  Once merged, update your local master again (``git pull --rebase upstream master``)
+8.  Tag the release with the new release version (``git tag v<new tag>``)
+9.  Push the tag (``git push upstream --tags``)
+10. Done. Check `CI <https://github.com/pytest-dev/pytest-html/actions>`_ for release progress.
```

Comment:
Here's a trick: You can change all the numbers to `1`, and RST will renumber them. That way, if you have to insert something, it will only be a one-line change. :)